publish/views/version.py

```diff
@@ -22,6 +22,8 @@ class Meta:
             'version',
             'created',
             'updated',
+            'count',
```

Comment:
Do these need to be listed as `read_only_fields`, or do we know that for free, since they're properties and not fields?

Reply:
Looks like they're read-only according to [these docs](https://docs.python.org/2/library/functions.html#property).